### ISSUE TITLE
A table header contains no text with filter row enabled

### ISSUE THREAD

**Issue by VicTachev** [telerik/kendo-ui-core#5314]
### Reproduction of the problem
When a Grid has filter row and command column the cell in the header contains no text. The error is reported by WAVE:

1. run this dojo - https://runner.telerik.io/fullscreen/eVIFOjUn
1. run WAVE tool - https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh

### Current behavior
![image](https://user-images.githubusercontent.com/11771009/66292272-974f6700-e8ec-11e9-951e-fb3d326ce130.png)

### Expected/desired behavior
There should be no error

### Environment

* **Kendo UI version:** 2019.3.917
* **jQuery version:** 1.12.2
* **Browser:** [all]

**Comment by Juveniel** [comment]
Also reported in TicketID: 1443359

**Comment by Dimitar-Goshev** [comment]
There is no error reported for this cell in version 2023.3.1114 - https://runner.telerik.io/fullscreen/@Dimitar-Goshev/owEBOSAL
